scrapy/core/engine.py

```diff
@@ -127,15 +128,16 @@ def _next_request(self, spider):
                 request = next(slot.start_requests)
             except StopIteration:
                 slot.start_requests = None
-            except Exception:
+            except Exception as exception:
                 slot.start_requests = None
                 logger.error('Error while obtaining start requests',
                              exc_info=True, extra={'spider': spider})
+                reason = str(exception)
```

Comment:
```suggestion
                reason = repr(exception)
```

https://youtu.be/V2fGAv2R5j8?t=9m2s

Reply:
@Gallaecio sure, working on it